src/patterns/gender-and-sex/index.md.njk

```diff
@@ -0,0 +1,50 @@
+---
+title: Gender and sex
+description: Understand when and how to ask users for their gender or sex.
+section: Patterns
+theme: Ask users for...
+aliases:
+backlog_issue_id: 69
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+Understand when and how to ask users for their gender or sex.
+
+## When to use this pattern
+
+Avoid asking users for their gender or sex.
+
+Only ask about gender or sex if you genuinely can’t deliver your service without this information.
+
+If you do need to ask, use ‘sex’ when you need biological data (for example, if you’re providing a medical service). In all other cases, use ‘gender’.
+
+## How it works
+
+If you have to ask about gender, you should:
+
+- always say why you’re asking and what you’ll do with the information
+- list the fields in alphabetical order, for example, ‘Female’, ‘Male’, ‘Nonbinary’
```

Comment:
are we sure about 'Nonbinary'? I can mostly see it written as 'Non-binary'

http://www.stonewall.org.uk/help-advice/glossary-terms

https://www.nhs.uk/Livewell/Transhealth/Pages/local-gender-identity-clinics.aspx

Reply:
I've looked online and also at GDS internal documents and yes, we use 'Non-binary' in most cases.

Reply:
I'm going to review and merge so that it's in the system for research, but we'll take another look and amend if necessary.